aiinfra-cluster/main.tf

```diff
@@ -88,7 +89,12 @@ module "nfs_filestore" {
 module "startup" {
   source          = "github.com/GoogleCloudPlatform/hpc-toolkit//modules/scripts/startup-script/?ref=1b1cdb09347433ecdb65488989f70135e65e217b"
   project_id      = var.project_id
-  runners = concat(local.dir_copy_setup
+  runners = concat(var.disable_ops_agent ? [] : [{
+    destination = "install_cloud_ops_agent.sh"
+    source      = "${path.module}/installation_scripts/install_cloud_ops_agent.sh"
+    type        = "shell"
+  }]
```

Comment:
NitPick(NP): For better readability maybe you can put this in locals block.

Reply:
Agreed, it follows the format of the vm_startup_scripts_setup variable nicely. Updated.